demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -221,41 +222,11 @@ def find_corresponding_yml(file_path):
     def identify_changed_file_type(self, file_path):
         _file_type = None
         file_name = 'N/A'
-        if 'ReleaseNotes' in file_path or 'TestPlaybooks' in file_path:
-            return file_name, _file_type
```

Comment:
This conditional is meant to handle a case where release notes were changed, or a test playbook was changed. Is this covered somewhere else?

Reply:
These files are filtered before with function filter_files_by_type - there is a global set called SKIP_RELEASE_NOTES_FOR_TYPES and these type are in it. So as I see and check these files will not reach this line.